### PR TITLE
[wip] Generate generic PageHeader and update css of time-entries

### PR DESCRIPTION
**Pull request by RikFrieling**

### CODE REVIEW

File: shared/components/page-header/PageHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    selectBoxes, displaySearchField
+  }
+) => (
+  <header className="page-header">
+    <section className="page-header__section">
+      <h1 className="page-header__title">
+        {text}
+      </h1>
+      <div className="page-header__divider" />
+      <h2 className="page-header__summation">
```
Comment (boris-arkenaar) [inline shared/components/page-header/PageHeader.jsx]: I would create one `h2` tag containing the count and the text.

File: shared/components/page-header/PageHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        {summation}
+      </h2>
+      <h2 className="page-header__summation-text">
+        {summationText}
+      </h2>
+    </section>
+    <section className="page-header__section">
+      {
+        !selectBoxes || selectBoxes.map(
```
Comment (boris-arkenaar) [inline shared/components/page-header/PageHeader.jsx]: I would stick to the convention of `selectBoxes && ...`.

File: shared/components/page-header/PageHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          (selectBox, index) => (
+            <SelectBox
+              key={`${index + selectBox.value}`}
```
Comment (boris-arkenaar) [inline shared/components/page-header/PageHeader.jsx]: Now the key varies according to the index and the selected value of the select box, that doesn't seem correct.

File: shared/components/page-header/PageHeader.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+              optionValues={selectBox.optionValues}
```
Comment (boris-arkenaar) [inline shared/components/page-header/PageHeader.jsx]: You could write
```
key={selectBox.idSomething}
{...selectBox}
```

**Comment by RikFrieling** [comment]
Changes requested above this comment are implemented

File: components/header/Header.jsx [hunk 5]
```diff
@@ -40,7 +40,7 @@ class Header extends React.Component {
               alt="cross"
             />
           </button>
-        </div>
+        </section>
         <NavBar isMenuOpen={isMenuOpen} toggleMenuOpen={toggleMenuOpen} />
```
Comment (jellebouwman) [inline components/header/Header.jsx]: I find it somewhat confusing that you import `HeaderNavBar`  as `NavBar` maybe stick to using one name?

File: components/time-entries/TimeEntries.jsx [hunk 6]
```diff
@@ -11,23 +11,23 @@ const TimeEntries = ({ timeEntries, onDelete }) => (
...
 
     return (
-      <React.Fragment key={timeEntry.id}>
+      <ul className="time-entries" key={timeEntry.id}>
         {(!index || date !== previousDate)
         && (
-          <div className="date-row">
-            <span>
+          <hgroup className="time-entries__time-entry-header">
+            <h1 className="time-entries__date">
```
Comment (jellebouwman) [inline components/time-entries/TimeEntries.jsx]: Maybe use `h2` and `h3` for this Date. I believe `h1` should only be used for the _biggest_ title!

**Pull request merged**